packages/shortest/src/ai/prompts/index.ts

```diff
@@ -42,6 +43,14 @@ IMPORTANT GLOBAL RULES:
    - All expectations listed in the test instructions must be fulfilled.
    - If any expectation is not met, the test case must be marked as failed.
 
+8. **Testing Email**:
+   - If you need to test a condition that involves seeing the contents of an email, use the "check_email" tool.
+   - For email validation, you MUST always use 'Click' and 'Mouse' action instead of using keyboard shortcuts.
+   - This tool will grab the latest email from the email address given to you and will render it in a new tab for you to see.
+   - Once you are done with validating the email, navigate back to the original tab.
+   - You MUST pass the email address that is given to you to the tool as a parameter otherwise it will fail.
+   - If no email address is given to you for this test, you should fail the test.
```

Comment:
Added a new prompt rule for Email Validation. Tried to stay consistent with the convention we have in place. 

We explicitly ask AI to use Click and Mouse events because sometimes AI might use tabs to focus on the next text filed that might result in focusing on something else. This would make the AI more reliable.